lib/cron.dart

```diff
@@ -55,6 +73,24 @@ class Schedule {
     if (days?.contains(time.day) == false) return false;
     if (weekdays?.contains(time.weekday) == false) return false;
     if (months?.contains(time.month) == false) return false;
+    final currentSecond = (seconds == null) ? seconds : time.second;
+    final currentMinutes = (minutes == null) ? minutes : time.minute;
+    final currentHour = (hours == null) ? hours : time.hour;
+    final currentDay = (days == null) ? days : time.day;
+    final currentMonth = (months == null) ? months : time.month;
+    final currentWeekday = (weekdays == null) ? weekdays : time.weekday;
+    if (currentSecond == lastSecond &&
```

Comment:
Please make the condition expression a separate variable and add comment on what that means...

```
// comment
final xyz = currentSecond == lastSecond && .... ;
if (xyz) {
  return false;
}

Reply:
```
    if ((seconds == null || lastTime.second == time.second) &&
        (minutes == null || lastTime.minute == time.minute) &&
        (hours == null || lastTime.hour == time.hour) &&
        (days == null || lastTime.day == time.day) &&
        (months == null || lastTime.month == time.month) &&
        (weekdays == null || lastTime.weekday == time.weekday)) {
      return false;
    }
```
I think there is no need to comment here, the condition means the whole logical. seconds equals null means any second can trigger cron, so no matter seconds equals null  or second match last datatime means the condition for the task has been ran.